src/processTargets/modifiers/LeadingTrailingStages.ts

```diff
@@ -22,10 +23,25 @@ export class TrailingStage implements ModifierStage {
   constructor(private modifier: TrailingModifier) {}
 
   run(context: ProcessedTargetsContext, target: Target): Target[] {
-    const trailing = target.getTrailingDelimiterTarget();
+    const trailing = getTargetToUse(
```

Comment:
if we just make it so that non-empty ranges aren't weak, then can't we just use a weak expander of token?